benchmark/src/main/resources/benchmark.xsd

```diff
@@ -317,6 +317,9 @@
         <xs:sequence>
                               
           
+          <xs:element maxOccurs="unbounded" minOccurs="0" name="enablePreviewFeatureList" nillable="true" type="tns:previewFeature"/>
```

Comment:
If I read this correctly, if there were more than one preview feature, this would look like this:

    <enablePreviewFeatureList>FEATURE_A</>
    <enablePreviewFeatureList>FEATURE_B</>

Then _for sure_ this shouldn't have the `list` suffix.